.github/workflows/weekly_update.yml

```
@@ -2,7 +2,7 @@ name: Weekly update
 
 on:
   schedule:
-    - cron: "0 0 * * 1"  # every Monday at midnight
+    - cron: "0 16 * * 1"  # every Monday at 08:00 PST (16:00 UTC)
```

At 8AM, there is likely no update to AMReX yet. So we can also do it earlier.

On Mondays, I would actually run it a bit earlier, e.g., before most developers get online, so CI is not running on this when people start their work.

How about we do 1AM Mondays (PT), which is still blocking the CI with this PR for European contributors, but not for US developers at least (which are currently the majority)?
```suggestion
    - cron: "0 9 * * 1"  # every Monday at 01:00AM PST (9:00AM UTC)
```